core/src/test/java/org/opensearch/sql/expression/datetime/ConvertTZTest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.expression.datetime;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+import static org.opensearch.sql.data.model.ExprValueUtils.missingValue;
+import static org.opensearch.sql.data.model.ExprValueUtils.nullValue;
+import static org.opensearch.sql.data.type.ExprCoreType.DATETIME;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.opensearch.sql.data.model.ExprDatetimeValue;
+import org.opensearch.sql.data.model.ExprValue;
+import org.opensearch.sql.expression.DSL;
+import org.opensearch.sql.expression.Expression;
+import org.opensearch.sql.expression.ExpressionTestBase;
+import org.opensearch.sql.expression.FunctionExpression;
+import org.opensearch.sql.expression.env.Environment;
+
+
+
+@ExtendWith(MockitoExtension.class)
+class ConvertTZTest extends ExpressionTestBase {
+
+  @Mock
+  Environment<Expression, ExprValue> env;
+
+  @Mock
+  Expression nullRef;
+
+  @Mock
+  Expression missingRef;
+
+  @BeforeEach
+  public void setup() {
+    when(nullRef.valueOf(env)).thenReturn(nullValue());
+    when(missingRef.valueOf(env)).thenReturn(missingValue());
+  }
+
+  @Test
+  public void convert_tz() {
```

Comment:
Each of the `convert_tz then assert` blocks is a separate test case and should be highlight as such.

The normal pattern is -- method name describes the test cases, the body of the method is arranges data, acts, then asserts. This test has multiple arrange, act, assert blocks. I'd like to quickly understand the cases they are testing.